src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList.tsx

```diff
@@ -24,11 +24,32 @@ const TaskList = ({ project, defaultClosed }: TaskListProps) => {
   const auth = useAuth();
   const [value, setValue] = useState<number>(1);
   const [addTask, setAddTask] = useState(false);
-
   const tasks = project.tasks;
-  const backLogTasks = tasks.filter((task: Task) => task.status === TaskStatus.IN_BACKLOG);
-  const inProgressTasks = tasks.filter((task: Task) => task.status === TaskStatus.IN_PROGRESS);
-  const doneTasks = tasks.filter((task: Task) => task.status === TaskStatus.DONE);
+
+  //used two sort two Tasks based on ascending times
+  const sortAscDate = (task1: Task, task2: Task) => {
+    const deadLine1 = task1.deadline.getTime();
+    const deadLine2 = task2.deadline.getTime();
+
+    if (deadLine1 < deadLine2) {
```

Comment:
```
if (deadline1 !== deadline2) return deadline1 - deadline2
return task1.datecreated.getTime() - task2.datecreated.gettime()
```

(make sure that this still does it in ascending order)

Reply:
done